src/Raven.Studio/wwwroot/App/views/resources/compactDatabaseDialog.html

```diff
@@ -17,6 +17,12 @@ <h3 class="modal-title">
                         <input id="allIndexes" type="checkbox" data-bind="checked: compactAllIndexes" />
                         <label for="allIndexes">Compact all indexes</label><small data-bind="text: numberOfIndexesFormatted"></small>
                     </div>
+                    <div class="margin-left margin-left-lg">
+                        <div class="toggle" title="Uncheck to skip optimization of Lucene indexes">
```

Comment:
Uncheck ?
Isn't it the opposite ?
e.g.: "Toggle on to skip optimization of Lucene indexes"

Reply:
You are right